pcweb/pages/customers/stories/bayesline.md

```diff
@@ -0,0 +1,106 @@
+---
+company: Bayesline
+description: "How Bayesline builds 4x faster using Reflex instead of Plotly Dash"
+domain: "https://bayesline.com/"
+founded: "New York, 2024"
+investors: "Y Combinator",
+meta: [
+  {
+    "name": "keywords",
+    "content": "
+      plotly dash,
+      python and react,
+      python and react js,
+      python react,
+      python web app,
+      python web app framework,
+      python web apps,
+      react js python,
+      react js with python,
+      react python,
+      react python websocket,
+      react with python,
+      web app framework python,
+      web app python,
+    "
+  }
+]
+---
+
+Why Bayesline chose Reflex instead of Plotly Dash
+
+## TLDR;
+
+Sebastian and his small team use Reflex to keep the full flexibility of a traditional ReactJS app **WITHOUT** needing to:
+
+* Learn frontend: React (JavaScript), NodeJS, TailwindCSS
+
+```md quote Misha
+Reflex definitely saved us from needing to hire a frontend engineer.
```

Comment:
I would recommend putting a dollar amount in here i.e. saved us $140,00 a year not needing to hire a front end engineer

Reply:
I dont think we can put a dollar amount on this as its not a hard number all frontend dev make x amount